src/pulse3D/excel_writer.py

```diff
@@ -196,6 +196,7 @@ def create_frequency_vs_time_charts(
 
 def write_xlsx(
     plate_recording: PlateRecording,
+    output_dir: str = os.getcwd(),
```

Comment:
I could be wrong here, but I think for handling default values like this it's better to set it to `None`, then in the function check if it's None and then set it to `os.getcwd()`. The reason for the being that `os.getcwd()` will only get called once and the value for it will be saved, so if you were to switch directories in the code somewhere it wouldn't use the new directory